File: bin/textnormalize.js
a = new ActiveXObject("scripting.filesystemobject")
fileinfo = ""
for (b = new Enumerator(a.GetFolder(WScript.Arguments(0)).files); !b.atEnd(); b.moveNext())
{
    f = a.opentextfile(b.item())
    try
    {
        shortf = a.GetBaseName(b.item()).toUpperCase()
        newf = ""
        fileinfo += "\r\n" + shortf + "=Type(04),AuxType(0000),VersionCreate(70),MinVersion(BE),Access(C3)"
        l = 0
        while (1)
        {
            lines = f.readline()
            !lines.length && (linelength = 0);
            l && linelength && (lines += new Array(linelength - lines.length).join(" "))
            newf += lines + "\r"
            if (lines.substring(0, 6) == "[info]") linelength = 65
            else if (lines.substring(0, 13) == "[description]") linelength = 78
            l = linelength
        }
    }
    catch(e){}
    a.createtextfile("build\\text\\" + shortf, 1).write(newf)
}
a.createtextfile("build\\text\\_FileInformation.txt", 1).write(fileinfo.substring(2))

File: bin/textnormalize.py
#!/usr/bin/env python3

import sys

fileinfo = []
for f in sys.argv[1:]:
    linelength = 0
    with open(f) as buffer:
        lines = buffer.readlines()
    shortf = f.replace(".txt", "").replace("res/text/", "").upper()
    newf = "build/TEXT/" + shortf
    fileinfo.append(shortf + "=Type(04),AuxType(0000),VersionCreate(70),MinVersion(BE),Access(C3)")
    with open(newf, 'w') as buffer:
        for l in lines:
            l = l[:-1]
            if not l:
                linelength = 0
            if l and linelength:
                l = l + ' '*(linelength-len(l))
            buffer.write(l + "\r")
            if l.startswith("[info]"):
                linelength = 64
            elif l.startswith("[description]"):
                linelength = 77
with open("build/TEXT/_FileInformation.txt", "w") as infobuff:
    infobuff.write("\r\n".join(fileinfo))
